### PR TITLE
Improvements to rw exec scripts 

### LINKED ISSUE #3279: Issue importing deeply nested files such as services via CLI Exec feature
There are two main issues i've come across with the CLI exec feature and how it's currently doing importing.

- Yarn workspace and shared packages cannot be used - there is an issue picking it
- Imports (relative and aliased) must be used with a lot of care. The exec feature needs to use `../api/src/....` where most of the codebase has an alias to use `src/...` which pick it up from `api/src`

### Reproduction Repo
https://github.com/viperfx/rw-cli-import-issue

- clone this repo
- install deps
- run yarn rw prisma migrate dev
- run either yarn rw exec aliasImport or yarn rw exec relativeImport
- Both produce a import error.

### Loom video discussing issues

https://www.loom.com/share/a270c72668ae4030902a2e0c3c7e8b9d

### Current Blockers
- This is currently preventing me from running background workers with my production application.
- One of scripts or scheduled scripts (run via repeater.dev) also also prevented from using deeply nested files in api.

### LINKED ISSUE #3372: Generating the prisma client option for CLI Exec
Right now the prisma client is always generated when running a `yarn rw exec` command. This should be possible to disable with a flag.

### PR DESCRIPTION
**Pull request by dac09**
Fixes #3279
Fixes #3372

## What does this do?
- Adds primsa option to cli
- Improves errors on script not found
- Adds overrides for babel module path settings, so that nested imports still work
- Generates a tsconfig in a typescript project, so that we get type/autocompletion/cmd+click even if we use the `$api/src/lib/bazinga` syntax

### CODE REVIEW

**Comment by dac09** [comment]
@viperfx thanks for the repro, speeded things up :) 

When are we getting that cookbook for how to use scripts/exec now? 😜

File: packages/cli/src/commands/generate/script/__tests__/script.test.ts [hunk 1]
```diff
@@ -26,12 +26,19 @@ test('creates a TypeScript function to execute', () => {
   const output = files({
     name: 'typescriptyTypescript',
     typescript: true,
-  })
+  }) //?
```
Comment (jtoar) [inline packages/cli/src/commands/generate/script/__tests__/script.test.ts:29]: Guessing this is a wallaby artifact right?
```suggestion
  })
```

**Review by jtoar — APPROVED** [review]
@dac09 LGTM, just one comment

**Comment by thedavidprice** [comment]
Added the "Breaking" label to remind us that folks should update their generated template (this will be necessary, correct)?

**Comment by dac09** [comment]
Nope, not necessary!

**Comment by viperfx** [comment]
> @viperfx thanks for the repro, speeded things up :)
> 
> When are we getting that cookbook for how to use scripts/exec now? 😜

I am on it!

**Comment by viperfx** [comment]
@dac09 PR for expanded usage of CLI scripts here - https://github.com/redwoodjs/redwoodjs.com/pull/815

Feedback welcome!

**Comment by thedavidprice** [comment]
YES I'll take a look

**Pull request merged**